application/src/main/java/org/opentripplanner/transit/service/TransitService.java

```diff
@@ -418,4 +418,14 @@ Collection<RegularStop> findRegularStopsByBoundingBox(
    * Returns a list of {@link StopLocation}s that match the filtering defined in the request.
    */
   Collection<StopLocation> findStopLocations(FindStopLocationsRequest request);
+
+  /**
+   * Returns boolean indicating if there are trips scheduled for a stop in the future.
+   */
+  boolean isStopInService(StopLocation stop);
```

Comment:
What about real-time updates?

Reply:
Javadoc should clearly state that realtime updates are not taken into account as well.